scripts/esCleaner.py

```diff
@@ -0,0 +1,31 @@
+import elasticsearch
+import curator
+import sys
+
+
+def main():
+    if len(sys.argv) == 1:
+        print('USAGE: %s HOSTNAME[:PORT] ...' % sys.argv[0])
+        sys.exit(1)
+
+    client = elasticsearch.Elasticsearch(sys.argv[1:])
+
+    ilo = curator.IndexList(client)
+    empty_list(ilo, 'ElasticSearch has no indices')
+
+    ilo.filter_by_age(source='name', direction='older', timestring='%Y-%m-%d', unit='days', unit_count=4)
```

Comment:
unit_count should be configurable. What if the company using this script has other indexes in elastic search? we don't want to delete those.